api/views/user.py

```diff
@@ -57,3 +57,21 @@ def login(request):
         return Response({'success': False, 'message': "Authentication credentials invalid", 'data': None}, status=status.HTTP_400_BAD_REQUEST)
     except:
         return Response({'success': False, 'message': "Authentication credentials invalid", 'data': None}, status=status.HTTP_400_BAD_REQUEST)
+
+@api_view(['PUT'])
+def edit_user(request, pk):
+    body_unicode = request.body.decode('utf-8')
+    body = json.loads(body_unicode)
+    name, sec = body['username'], body['secret']
+
+    try:
+        user = User.objects.get(id=pk)
+
+        if name is not "":
+            user.username = name
+        elif sec is not "":
+            user.secret = make_password(sec)
```

Comment:
So this route is so we can enter the text password instead of the hashed password when editing user right?